### DISCUSSION TITLE
Feature request: add ZoneMinder, Viseron, Frigate, OpenRemote, Dolibarr ERP & CRM, Odoo.

### DISCUSSION THREAD

**Discussion by 391a** [tteck/Proxmox#1547] [Ideas]
https://github.com/ZoneMinder/ZoneMinder/
https://github.com/roflcoopter/viseron
https://github.com/blakeblackshear/frigate
https://github.com/openremote/openremote
https://github.com/dolibarr
https://github.com/odoo/odoo

**Comment by tteck** [comment]
Double posted
